pom.xml

```diff
@@ -114,6 +114,56 @@ under the License.
             <version>${powermock.version}</version>
             <scope>test</scope>
         </dependency>
+      <dependency>
```

Comment:
I first tried using only `crail-client`, but `ClassNotFoundException` occurs. I am not sure why it works this way, so I just put all modules Crail provides (in case the specific module is needed). 
:(